documentation/sphinx/source/command-line-interface.rst

```diff
@@ -416,6 +416,13 @@ The ``set`` command sets a value for a given key. Its syntax is ``set <KEY> <VAL
 
 Note that :ref:`characters can be escaped <cli-escaping>` when specifying keys (or values) in ``fdbcli``.
 
+renametenant
```

Comment:
This document is at least mostly in alphabetical order, so let's move this up accordingly.